src/test/java/ttt/PlayerFake.java

```diff
@@ -0,0 +1,25 @@
+package ttt;
+
+import ttt.game.Board;
+import ttt.game.Marks;
+
+public class PlayerFake implements Player {
+
+    private Marks mark;
+
+    public PlayerFake(Marks mark) {
+        this.mark = mark;
+    }
+
+    public Marks getMark() {
+        return mark;
+    }
+
+    public int getLocation(Board board) throws Exception {
+        return 0;
+    }
+
+    public Class playerType() {
+        return getClass();
+    }
+}
```

Comment:
Uses a player fake so that tests can be agnostic to web or console human players